src/main/java/core/basesyntax/strategy/TransactionsHandler.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.model.Transaction;
+
+public interface TransactionsHandler {
+    void applyTransaction(Transaction transaction);
```

Comment:
```suggestion
    void handle(Transaction transaction);
```
handler handles something